.gitignore

```diff
@@ -1 +1,2 @@
 dist
+.idea
```

Comment:
This is editor specific, please consider adding it to your private global gitignore: https://blog.sebastian-daschner.com/entries/global-gitignore instead :)

Reply:
Great advice. Thanks!